src/markdown.cpp

```diff
@@ -3055,15 +3055,6 @@ QCString Markdown::processBlocks(const QCString &s,const int indent)
   int i=0,end=0,pi=-1,ref,level;
   QCString id,link,title;
 
-  // get indent for the first line
```

Comment:
why is this whole section removed? `end` is still used at line 3078? maybe you meant to remove only `sp` usage?

Reply:
You meant the usage in the lines:
```
  while (i<size)
  {
    findEndOfLine(data,size,pi,i,end);
    // line is now found at [i..end)

    int lineIndent=0;
    while (lineIndent<end && data[i+lineIndent]==' ') lineIndent++;
...
```
yes it is used here but when we look in `findEndOfLine` the first statements here are:
```
{
  TRACE(data);
  // find end of the line
  int nb=0;
  end=i+1;
```
so it `end` is reset.
Actually the declaration of end could take place inside the while loop.
(I first removed the `sp` and had at that moment also a look at `end` and saw what is written above, so I removed the complete loop).

Reply:
Ah you're right, I see it now.